nova/api/openstack/compute/servers.py

```diff
@@ -632,6 +632,11 @@ def create(self, req, body):
                 exception.NoUniqueMatch,
                 exception.MultiattachSupportNotYetAvailable) as error:
             raise exc.HTTPConflict(explanation=error.format_message())
+        except exception.InvalidQuotaMethodUsage as error:
+            if ' instances_' in error.message:
```

Comment:
It's not readily apparent to me why this would apply only to baremetal instances. In various places `'instances_'` is prepended to the flavor name, not only baremetal ones.